DOCS/man/mpv.rst

```diff
@@ -159,6 +159,9 @@ L
 Ctrl + and Ctrl -
     Adjust audio delay (A/V sync) by +/- 0.1 seconds.
 
+Shift+g and Shift+f
+    Adjust subtitle font size by +/- 0.1.
```

Comment:
Wouldn't "increase subtitle scaling by +/- 10%" be more correct?

Reply:
Done